src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php

```diff
@@ -160,6 +159,21 @@ public function iSaveScreenshot($fail = false)
         }
     }
 
+    /**
+     * Save screenshot with specific dimensions.
+     *
+     * @param int $width  Width to resize browser to.
+     * @param int $height Height to resize browser to.
+     *
+     * @When save :width x :height screenshot
+     * @When I save :width x :height screenshot
+     */
+    public function iSaveSizedScreenshot($width = 1440, $height = 900)
+    {
+        $this->getSession()->resizeWindow((int) $width, (int) $height, 'current');
```

Comment:
@xurizaemon 
Could you please put a check if the current session support window resize. With how this code currently written, using this step in non-JS enabled feature would trigger an exception.

Similar to https://github.com/integratedexperts/behat-screenshot/blob/master/src/IntegratedExperts/BehatScreenshotExtension/Context/ScreenshotContext.php#L134

Reply:
Done. (Is there a better method to check driver capabilities than catching UnsupportedDriverActionException as I've done here?)